collections/galaxy.yml

```diff
@@ -0,0 +1,63 @@
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: vmware
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: ansible_for_nsxt
```

Comment:
Wouldn't just "nsxt" be a nicer option, as ansible is fairly implied in an ansible collection?

Reply:
- ansible_for_nsxt matches this repo name so I think it'd be more consistent
- There already exists an older version on galaxy which we can reuse: https://galaxy.ansible.com/vmware/ansible_for_nsxt